src/test/java/Hw21.java

```diff
@@ -0,0 +1,17 @@
+import org.testng.annotations.Test;
+
+public class Hw21 extends BaseTest {
+    @Test
+    public void createPlaylistTest() {
+        logIn("demotesting@gmail.com", "te$t$tudent");
+        createPlaylist();
+        playlistSelect();
+        enterNameofplaylist();
```

Comment:
Please,use Assertions In public void createPlaylist()

For example  `Assert.assertTrue(notificationText());`

```
 public boolean notificationText() {
        return wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".success"))).isDisplayed();
    }
```